src/components/debug/syncAsync.js

```diff
@@ -8,23 +8,15 @@ const Container = styled.div`
   justify-content: space-around;
 `;
 
-const SyncAsync = ({ async }) => {
-  const toSync = () => {
-    window.location.href = "/";
-  };
-  const toAsync = () => {
-    window.location.href = "/?async";
-  };
-  return (
-    <Container>
-      <button disabled={!async} onClick={toSync}>
-        Normal rendering
-      </button>
-      <button disabled={async} onClick={toAsync}>
-        Concurrent rendering
-      </button>
-    </Container>
-  );
-};
+const SyncAsync = ({ async, setAsync }) => (
+  <Container>
+    <button disabled={!async} onClick={() => setAsync(false)}>
```

Comment:
Isn't it a problem without useCallback ? I mean, recreating a function on each render?

Reply:
Danny clearly mention (don't remember where) that unless you have real performance issues, it's not a problem.

To go a bit further, if I understand well, using `useCallback` will not be really better as passing the function in parameter will also create un function each time even if it's not to use it. As I see it, only a `useMemo` returning a function will really prevent to create a function at each render.